getting_started/data_backup.md

```diff
@@ -0,0 +1,74 @@
+9. You will get a link at which you should authenticate with your Google account. **At this step, be sure to use your Andrew account if that's where you want to back up!"** You may get a warning, proceed anyway (in a Chrome browser, by clicking Advanced and then "go to rclone"). 
+10. After authenticating, copy the code and paste into the rclone prompt.
+11. From here, you should be able to continue with default options and finish configuring! Phew!
+
+#### Box
+5. Follow default options until the "use auto config?" prompt. Because we are on a remote machine, you should enter `n` for this. 
+6. You will then need to open a local terminal session to access your web browser for authentication. Follow the instructions to do so.
+7. After authentication, on your local terminal, you should get something to paste into your remote session. Make sure to copy everything between the arrows.
+8. Follow defaults to finish configuration.
+
+### Running an `rclone` backup
+You will probably want to create a folder on your remote (GDrive or Box) in which your backups will live. Let's suppose you've done that and it's called `arjuna_backup`, and that the name of your remote is `my_remote` (if you don't remember what you called it, run `rclone listremotes`).
+
+Assuming you are in your home directory and you want to back up the entire thing, the syntax would then be
+```bash
+rclone sync -P . my_remote:arjuna_backup
+```
+The `-P` flag isn't necessary, but it will give you progress updates. You may get a bunch of warnings about symlinks not being copied.
+
+Note that for your first backup, if you have a lot of stuff in your directory, it may take a long time to run, and could get interrupted if your internet connection cuts out. `tmux` is a useful tool to get around this and keep tasks running even if you disconnect (many good tutorial easily found on Google).
+
+### Scheduling backups
+An easy "set-it-and-forget-it" approach for backups is to schedule them using cron jobs. A simple setup for this can be achieved by:
+1. Write a bash script to run `rclone` on whatever files/directories you'd like to sync. Let's assume this is called `backup.sh` (the simplest version would be a single-line script containing only the command above to sync your entire home directory, but you could include multiple lines to, for example, skip directories containing only large number of data files and instead sync a single tarball containing these files).
+2. Run the command `crontab -e` to open your cron table, this should open a file in your editor which will be blank if you've never used cron before.
+3. Add a line like `0 2 * * * /home/username/backup.sh` to this file, replacing the path with the appropriate path to your backup script. This line will run the backup every day at 2am. For syntax of these schedulers, see e.g. [this page](https://crontab.guru).
```

Comment:
Let's change this to monthly for the default schedule. Full daily backups (As written this backs up everything) could reach 350 - 500 GB / day / user. Arjuna does not have the bandwidth for that.